relayer/aztec/relayer.go

```diff
@@ -20,44 +18,72 @@ import (
 	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethereum/go-ethereum/ethclient"
 	vaaLib "github.com/wormhole-foundation/wormhole/sdk/vaa"
+	"go.uber.org/zap"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials/insecure"
 )
 
+// Global logger
+var logger *zap.Logger
+
+// Initialize global logger
+func initLogger() {
+	var err error
+
+	// Check if we're in development mode
+	devMode := os.Getenv("DEV_MODE") == "true"
+	if devMode {
+		// Development logger - more human-readable
+		logger, err = zap.NewDevelopment()
+	} else {
+		// Production logger - JSON format, better for log aggregation
+		logger, err = zap.NewProduction()
+	}
+
+	if err != nil {
+		// Fallback to standard logger if zap fails
+		fmt.Printf("Failed to initialize zap logger: %v\n", err)
+		logger = zap.NewExample()
+	}
+
+	// Ensure logger is flushed at program exit
+	defer logger.Sync()
```

Comment:
If I'm reading the code correctly, this will run right away, not at program exit.